src/puppetlabs/trapperkeeper/services/webrouting/webrouting_service_core.clj

```diff
@@ -101,6 +101,16 @@
                                                                  route-id)]
     (:route endpoint-and-server)))
 
+(schema/defn ^:always-validate get-server
```

Comment:
can you add schema for the return type:  Just glancing at the code it's hard to tell whether it's a server id or an actual Jetty `Server` object.